public/request/game/recalculate-points-ratio.php

```diff
@@ -11,7 +12,9 @@
     'game' => 'required|integer|exists:mysql_legacy.GameData,ID',
 ]);
 
-if (recalculateTrueRatio((int) $input['game'])) {
+/** @var UpdateGameWeightedPointsAction $updateGameWeightedPointsAction */
+$updateGameWeightedPointsAction = app()->make(UpdateGameWeightedPointsAction::class);
+if ($updateGameWeightedPointsAction->run((int) $input['game'])) {
```

Comment:
Not sure what's going on, but attempting to Recalculate True Ratio from a game page is failing

Reply:
Seems the casing of the `$gameId` variable got messed up. Fixed